### ISSUE TITLE
Jenkins 2.0 support with Jenkinsfile

### ISSUE THREAD

**Issue by shravanjnaidu** [danger/danger#961]
# Report

## What did you do?

I'm trying to run danger on Pull request on Jenkins 2.0 with Jenkinsfile but I don't have ghprb set. Instead of git hooks I'm using the `Multibranch pipeline` which scans my github org for changes.

I expect it to run danger without hooks set up. Is that possible with danger?

## What happened instead?

It did not see my PR as PR and prints a message `Not a Pull Request - skipping danger run`

## Your Environment

* Which CI are you running on? Jenkins 2.X
* Are you running the latest version of Danger? Yes

**Comment by orta** [comment]
I think it's worth looking into adding support yourself for it, I don't know much about it (and just use Travis, personally) but will happily accept and maintain PRs adding it

**Comment by HannesT117** [comment]
Hi shravanjnaidu,
have you declared `ghprbGhRepository=(not quite sure)` and `ghprbPullId="${env.CHANGE_ID}"` as environment variables in your jenkinsfile? At least this was an issue for me using BitBucket.

**Comment by duncwa** [comment]
Yes, @shravanjnaidu, try @HannesT117 solution ```ghprbPullId = "${env.CHANGE_ID}"``` . I would like to add that we also set ```PR_URL = "${env.CHANGE_URL}"```

**Comment by duncwa** [comment]
One other issue that I have yet to resolve is the fact that a danger lane (fastlane) does not send the markdown message updates of the tests results to the bitbucket server PR.  Have yet to resolve this issue.  Only fails in MBP (MultiBranchPipeline) but not in regular Jenkins pipeline runs.
